packages/backend/server/src/plugins/payment/service.ts

```diff
@@ -46,20 +47,20 @@ const OnStripeEvent = (
 export function encodeLookupKey(
   plan: SubscriptionPlan,
   recurring: SubscriptionRecurring,
-  variant?: SubscriptionPriceVariant
+  variant?: SubscriptionVariant
 ): string {
   return `${plan}_${recurring}` + (variant ? `_${variant}` : '');
 }
 
 export function decodeLookupKey(
   key: string
-): [SubscriptionPlan, SubscriptionRecurring, SubscriptionPriceVariant?] {
+): [SubscriptionPlan, SubscriptionRecurring, SubscriptionVariant?] {
   const [plan, recurring, variant] = key.split('_');
 
   return [
     plan as SubscriptionPlan,
     recurring as SubscriptionRecurring,
-    variant as SubscriptionPriceVariant | undefined,
+    variant as SubscriptionVariant | undefined,
   ];
```

Comment:
it's kinda meaningless because the prices are consistent configured in stripe and the app will fail if the prices are setting wrongly. we can't recover from any unexpected input. user facing types are already protected by GraphQL